accounts/abi/argument.go

```diff
@@ -33,24 +33,26 @@ type Argument struct {
 
 type Arguments []Argument
 
+type ArgumentMarshaling struct {
+	Name    string
+	Type    string
+	Indexed bool
+}
+
 // UnmarshalJSON implements json.Unmarshaler interface
 func (argument *Argument) UnmarshalJSON(data []byte) error {
-	var extarg struct {
-		Name    string
-		Type    string
-		Indexed bool
-	}
-	err := json.Unmarshal(data, &extarg)
+	var arg ArgumentMarshaling
+	err := json.Unmarshal(data, &arg)
 	if err != nil {
 		return fmt.Errorf("argument json err: %v", err)
 	}
 
-	argument.Type, err = NewType(extarg.Type)
+	argument.Type, err = NewType(arg.Type, nil)
```

Comment:
Shouldn't it be like below?
```suggestion
	argument.Type, err = NewType(arg.Type, arg.Components)
```

Reply:
Applied at https://github.com/klaytn/klaytn/pull/792/commits/67e436bb5dbf4b2a9170cff0118131f382656894